task_13/Aleksandr.Timoshenko/src/test/java/com/hillel/complex/TestComplexNumber.java

```diff
@@ -0,0 +1,55 @@
+package com.hillel.complex;
+
+import static org.hamcrest.core.Is.is;
+import static org.junit.Assert.assertThat;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class TestComplexNumber {
+
+  ComplexNumber firstComplexNumber;
+
+  @Before
+  public void setUp() {
+    firstComplexNumber = new ComplexNumber();
+
+  }
+
+  @Test
+  public void testAddition() {
+    firstComplexNumber.setComplexNumber(-1, -3);
+    ComplexNumber complexResult = firstComplexNumber.addition(new ComplexNumber(3, 2));
+    assertThat(true, is(complexResult.equals(new ComplexNumber(2, -1))));
```

Comment:
It should be:
assertThat(complexResult.equals(new ComplexNumber(2, -1)), is(true));

Correct it in all places